### PR TITLE
Converted half of rooms into correct spawning format and correct badd…

### PR DESCRIPTION
**Pull request by whitew1994WW**
…ie type

### CODE REVIEW

**Comment by Jamesbarford** [comment]
Seems strange to add the more buffed baddies to the spawn points?

File: Moonshot/room_templates/room_scenes/RouteRooms/room_DLR_JumpingGallery.tscn [hunk 1]
```diff
@@ -8,6 +8,12 @@
 [ext_resource path="res://room_templates/exit_hori.tscn" type="PackedScene" id=6]
 [ext_resource path="res://assets/tile_maps/TileSet_v1.tres" type="TileSet" id=7]
 
+[ext_resource path="res://baddies/characters/FlyBoi.tscn" type="PackedScene" id=8]
+[ext_resource path="res://baddies/characters/GhostyBoi.tscn" type="PackedScene" id=9]
+[ext_resource path="res://baddies/characters/TankyBoi.tscn" type="PackedScene" id=10]
+[ext_resource path="res://baddies/characters/BearBoi.tscn" type="PackedScene" id=11]
+[ext_resource path="res://baddies/characters/Chick.tscn" type="PackedScene" id=12]
+
```
Comment (wattsit) [inline Moonshot/room_templates/room_scenes/RouteRooms/room_DLR_JumpingGallery.tscn:16]: More Enemies?

**Comment by wattsit** [comment]
image


I don't really understand whats happening here. I thought if you left the spawn points and lines empty it would pick random enemies?

**Comment by whitew1994WW** [comment]
- All of therooms that need to have the difficulty scaling need to have spawn points. 
- Each spawn point will choose random baddies if you don't specify then it will choose a random baddie from the complese list of baddies. However, there are very few spawn points that are suitable for every baddie type wether its locationis on a wall or the ceiling

**Pull request closed without merging**